data_classes/lib/data_classes.dart

```diff
@@ -21,6 +24,7 @@ class DataClass {
   const DataClass({
     this.builtValueSerializer = false,
     this.childrenListener,
+    this.convertToSnakeCase = false,
```

Comment:
Should not it be on by default?

Reply:
Why should it be on? We use camelCase in our json files by default